CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationActions.uc

```diff
@@ -39,12 +39,12 @@ static function X2CovertInfiltrationTemplate CreateInfiltrationTemplate(name Cov
 
 	if (bCreateSlots)
 	{
+		Template.Slots.AddItem(CreateDefaultOptionalSlot('InfiltrationStaffSlot'));
```

Comment:
What's the point of this change?

Reply:
Oh, whoops. That probably shouldn't be in this PR. Basically it just moves the optional slots to be on either side with the required four in the middle, like vanilla. How it was setup before, both optional slots were together on the right side.